app/javascript/components/search/controls/download/DownloadButton.js

```diff
@@ -56,7 +70,26 @@ export default function DownloadButton({ searchResults={} }) {
         </button>
       </OverlayTrigger>
       { showModal &&
-        <DownloadSelectionModal show={showModal} setShow={setShowModal} studyAccessions={matchingAccessions}/> }
+        <DownloadSelectionModal
+          show={showModal}
+          setShow={setShowModal}
+          tdrFileInfo={tdrFileInfo}
+          studyAccessions={matchingAccessions}/> }
     </>
   )
 }
+
+/** limit the files returned to 10, and add two fake analysis files
+  */
+function makeDemoAppropriate(fileInfomation) {
+  return fileInfomation.slice(0, 10).concat(FAKE_ANALYSIS_FILES)
+}
+
+/** these are real (sequence file) DRS ids */
+const FAKE_ANALYSIS_FILES = [{
```

Comment:
One more NIT on naming, the comment says these are real DRS ids, so calling them `FAKE_ANALYSIS_FILES` feels funny, wondering if maybe `DEMO_ONLY_ANALYSIS_FILES` or something might fit better?